### ISSUE TITLE
Reporte con nombre obsoleto

### ISSUE THREAD

**Issue by nepito** [IslasGECI/reproducibility_inspector#11]
Después de [cambiar el nombre de un reporte en el `analyses.json`](https://bitbucket.org/IslasGECI/conejos_clarion/commits/2841e2e929985b44633c6ac505a7177e77c8d61c) en la mañana apareció en rojo el intento por correr el reporte con el nombre viejo en la rama _develop_. 

Para resolverlo borramos `testmake.test.csv` en `api-tablero`. La mañana siguiente volvió aparecer ese mismo reporte fallando, así que lo borramos del `tesmake.log.csv`.

No sabes:
- De dónde, el inspector tomó el nombre viejo del reporte.

**Comment by nepito** [comment]
Sí, justo eso pensamos. Ojalá que vuelva a fallar para poder mejorar la descripción del _issue_.

**Comment by nepito** [comment]
Por tercer día consecutivo el `reports/dendidades_lineales_conejos.pdf` aparece tronando:
![image](https://user-images.githubusercontent.com/35377740/95599606-26aebf00-0a06-11eb-8fa0-d1da38cae5be.png)
El día martes cerramos el _pull request_ que incluía el cambio del nombre del reporte. A partir de ahí el inspector no ha actualizado el nombre del reporte que debería de tratar de generar.

Pareciera que el inspector guardó el nombre de los reporte que debe generar y no ha actualizado esa lista.

**Comment by evaristor** [comment]
La instrucción `git checkout origin/develop -- analyses.json` no está funcionando como lo esperamos:

![image](https://user-images.githubusercontent.com/3461018/95613038-2bc93980-0a19-11eb-92c5-db598e736baf.png)

**Comment by evaristor** [comment]
Es un problema con las credenciales:
![image](https://user-images.githubusercontent.com/3461018/95613717-3df7a780-0a1a-11eb-82ac-ec9589643876.png)

**Comment by evaristor** [comment]
El problema está en que las credenciales se quedan guardadas en la ruta del `origin` al momento de clonar el repo.
Si el repo existe (ya fue clonado) hacemos _fetch_ con las mismas credenciales que usamos para clonar el repo. Si el repo no existe, entonces lo clonamos y guardamos las credenciales para ser usadas en el futuro.

![image](https://user-images.githubusercontent.com/3461018/95614539-85326800-0a1b-11eb-9fc5-c48d360e4ec4.png)

**Comment by evaristor** [comment]
Temporalmente, se resuelve eliminando todos los repos clonados que se encuentran en el volumen `reproducibility_inspector_vol` depués de cambiar las credenciales de Bitbucket.
